packages/frontend/web-ui/src/game/components/FinishedGameListItem.tsx

```diff
@@ -0,0 +1,35 @@
+import { models as apiModels } from '@cornie-js/api-models';
+import { Box, Typography } from '@mui/material';
+
+import {
+  useGetFinishedGameWinner,
+  UseGetFinishedGameWinnerResult,
+} from '../hooks/useGetFinishedGameWinner';
+import { BaseGameListItem } from './BaseGameListItem';
+
+export interface FinishedGameListItemOptions {
+  game: apiModels.GameV1;
+}
+
+function getWinner(
+  game: apiModels.GameV1,
+): UseGetFinishedGameWinnerResult | undefined {
+  if (game.state.status === 'finished') {
+    return useGetFinishedGameWinner(game);
+  } else {
+    return undefined;
+  }
+}
+
+export const FinishedGameListItem = (options: FinishedGameListItemOptions) => {
+  const nameGameWinner: string | undefined = getWinner(options.game)
```

Comment:
The current solution looks good, but we should avoid requesting winners for every single game to render. Instead of this, we should get all the winners in a single call in the same way the `PublicGames` page fetches games specs with a single HTTP call (consider having a look at `useGetGamesWithSpecsV1`).